.gitignore

```diff
@@ -2,3 +2,7 @@
 /side.js
 /qodana.yaml
 /.idea
+
+/dockerfile
+/nginx.conf
```

Comment:
What are these for?

Reply:
Self Hostable Version is in development. I am working on it locally for now.